cvat/apps/engine/data_manager.py

```diff
@@ -290,6 +290,9 @@ def _modify_unmached_object(obj, end_frame):
             shape["frame"] = end_frame
             shape["outside"] = True
             obj["shapes"].append(shape)
+            # Need to reset cached interpolated shapes
+            # because key shapes were changed
+            obj['interpolated_shapes'] = []
```

Comment:
@azhavoro , is it a performance issue from your point of view? Just want to understand if the solution is good or bad. In general there are two possible solutions here:
- `obj['interpolated_shapes'] = []`
- interpolate till the end (in this case it is trivial because just need to copy the shape till end_frame).

Reply:
Let's use the interpolation approach, updated.